### PR TITLE
Change term TagRule to GlobalFilter

### PR DESCRIPTION
**Pull request by Freakachoo**
According to issue https://github.com/curiefense/curiefense/issues/552 the term `TagRule` was changed to `GlobalFilter`.

# Changelog
The term was changed for:
- UI
- URL Paths
- CurieConf and CurieProxy variables and functions names
- Filenames (json schema, config, initial_db)
- Kibana Dashboard (Blocked by TagRules -> Blocked by GlobalFilter)

Also, JSON schemas were divided into `v1` and `v2` and moved under corresponding folders.
Because the internal logic of a new term and API v2 was not changed (only name), function `vconvert` was created to convert between `v1` and `v2` terms.

# TODO
Update docs

### CODE REVIEW

**Review by bartavelle — APPROVED** [review]

**Review by flaper87 — CHANGES_REQUESTED** [review]
curietasker is failing to build:

```
2021-08-12T10:11:43.8576051Z Step 4/12 : COPY curieconf-server/curieconf/confserver/json/tag-rules.schema /tag-rules.schema
2021-08-12T10:11:43.8616803Z COPY failed: file not found in build context or excluded by .dockerignore: stat curieconf-server/curieconf/confserver/json/tag-rules.schema: file does not exist
```

Also, do you think we can update the `e2e` tests so that they won't fail :pray: 

Thanks for the work here :+1:

File: .github/workflows/e2e-minikube-filebeat.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
...
...tby_authority or test_ratelimit_event_uri or test_ratelimit_event_path or test_ratelimit_event_query or test_ratelimit_event_method or test_ratelimit_event_authority or test_ipv6 or test_country or test_asn or test_ratelimit_scope_include or test_ratelimit_scope_include_exclude or test_ratelimit_scope_exclude or test_ratelimit_countby_section or test_ratelimit_countby2_section or test_ratelimit_countby_2sections or test_ratelimit_event_section or test_length_overlong or test_length_short or test_count_few or test_count_toomany or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_allowlisted_value or test_non_allowlisted_value_restrict or test_non_allowlisted_value_norestrict_nowafmatch or test_non_allowlisted_value_norestrict_wafmatch or test_non_allowlisted_value_norestrict_wafmatch_excludesig or test_allowlisted_value or test_allowlisted_value or test_non_allowlisted_value_restrict or test_non_allowlisted_value_norestrict_nowafmatch or test_non_allowlisted_value_norestrict_wafmatch or test_non_allowlisted_value_norestrict_wafmatch_excludesig)' .
```
Comment (flaper87) [inline .github/workflows/e2e-minikube-filebeat.yml:62]: For a follow-up PR: Given that the compose tests pass, I wonder if we should try to enable all these for the istio+envoy based ones. It should be the same.

**Review by flaper87 — COMMENTED** [review]
This is fantastic, @Freakachoo 

I would ask @bartavelle to take a look, given the changes to the Rust part too.

**Review by flaper87 — APPROVED** [review]
Please, use squash as the merge strategy

**Review by bartavelle — APPROVED** [review]

File: .github/workflows/e2e-minikube-filebeat.yml (hunk 1 above)
Comment (Freakachoo) [reply .github/workflows/e2e-minikube-filebeat.yml:62]: @yanushg ^^^ please consider adding this on a new tests implementation

Comment (Mamba1981) [reply .github/workflows/e2e-minikube-filebeat.yml:62]: These tests will be enabled in new test implementation, thanks.

**Pull request merged**